Mathlib/MeasureTheory/Constructions/BorelSpace/Order.lean

```diff
@@ -462,23 +544,23 @@ theorem measurableSet_uIoc : MeasurableSet (uIoc a b) :=
 
 variable [SecondCountableTopology α]
 
-@[fun_prop]
+@[measurability, fun_prop]
```

Comment:
The `measurability` tactic now calls `fun_prop`, so I think the tags you add here don't do anything?